assembl/static2/js/app/components/home/phases/phase.jsx

```diff
@@ -38,20 +59,33 @@ class Phase extends React.Component {
   }
 
   render() {
-    const { imgUrl, startDate, title, description, index } = this.props;
+    const { imgUrl, startDate, title, description, index, screenWidth } = this.props;
     const stepNumber = index + 1;
+    const backgroundImage = imgUrl ? `url(${imgUrl})` : null;
+
+    // Return top margin value according to the screenwidth
+    const boxTopMargin = () => {
+      if (screenWidth < SM_SCREEN_WIDTH) {
+        const { maxTopMargin, marginStep, phasesPerLine } = BOX_TOP_MARGIN.mobile;
+        return maxTopMargin - (index % phasesPerLine) * marginStep;
+      } else if (screenWidth >= SM_SCREEN_WIDTH && screenWidth < LG_SCREEN_WIDTH) {
```

Comment:
These conditions should be functions, like `isMidsizedScreen` or something of that nature. There is also the `isMobile()`, perhaps can add these functions to the prototype of this object.

Reply:
updated !